monitoring-alert-rules/README.md

```diff
@@ -4,49 +4,51 @@
 
 This example shows how to configure alert rules in Kyma.
 
-1. Define a new alerting rule in Prometheus.
-2. Configure Alertmanager to handle alerts.
+- Define a new alert rule for AlertManager.
 
 ## Prerequisites
 
-- Review [alert-rules/README.md](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/monitoring/charts/alert-rules/README.md) in Kyma. It is **mandatory** in order to complete the example.
-- Review [alertmanager/README.md](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/monitoring/charts/alertmanager/README.md) in Kyma. It is **mandatory** in order to complete the example.
 - Kyma as the target deployment environment.
-- VictorOps Routing Keys and a Service API Key [Prometheus Integration Guide – VictorOps](https://help.victorops.com/knowledge-base/victorops-prometheus-integration/)
 
 ## Installation
+> Note: You need access to the `kyma-system` Namespace to execute the following steps
 
-1. Edit  [unhealthy-pods-rules.yaml](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/monitoring/charts/alert-rules/templates/unhealthy-pods-rules.yaml) by removing the line comment symbol ```#```
-2. Edit [alertmanager/values.yaml](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/alertmanager/values.yaml) to enable the victorOps (and/or) slack integration. **You will have to replace some data such a tokens and urls with your configuration**.
-3. Run Kyma followed by the kubectl command to access prometheus dashboard.
-    ```$ kubectl port-forward pod/prometheus-core-0 -n kyma-system 9090:9090```
-    In [http://localhost:9090/rules](http://localhost:9090/rules), you will find the rule, **pod-not-running-rule**.
-    ![pod-not-running-rule](images/pod-not-running-rule.png)
+### Configure a new alert
+1. Create a ConfigMap for the alert-rule.
 
-4. After Kyma in fully running we should start to receive alerts in VictorOps or Slack.
-- VictorOps
-![VictorOps alerts](images/victorops-alert.png)
+    ```bash
+    kubectl apply -f deployment/alert-rule-configmap.yaml -n kyma-system
+    ```
 
-- Slack
-![Slack alerts](images/slack-alert.png)
+2. Run the `port-forward` command on the `core-prometheus` service to access the prometheus dashboard.
+    
+    ```bash
+    kubectl port-forward pod/prometheus-core-0 -n kyma-system 9090:9090
+    ```
 
-**Review you configuration if the alerts are not received after the expected time. You also should Validate ```alertmanager/values.yaml``` files.**
+    In [`http://localhost:9090/rules`](http://localhost:9090/rules), you will find the rule, **http-db-service-is-not-running**.
```

Comment:
``Find the **http-db-service-is-not-running** rule in `http://localhost:9090/rules`](http://localhost:9090/rules).`` -> if we can hide this link it would be great but if not, I won't argue

Reply:
link is hidden now